Description: Disable language-specifications testsuite
 Disable language-specifications testsuite which requires network access.
Author: Ilias Tsitsimpis <iliastsi@debian.org>

--- a/mustache.cabal
+++ b/mustache.cabal
@@ -92,6 +92,7 @@
 test-suite language-specifications
   type: exitcode-stdio-1.0
   main-is: Language.hs
+  buildable: False
   other-modules:
       Paths_mustache
   hs-source-dirs:
